### DISCUSSION TITLE
Has the "sort by" been removed in PnP-Modern-Search V4?

### DISCUSSION THREAD

**Discussion by meklulose** [microsoft-search/pnp-modern-search#1560] [Ideas]
I remember "sort by" in PnP-Modern-Search V3 but I can't see this option in PnP-Modern-Search V4.
![image](https://user-images.githubusercontent.com/81180944/143471330-8554f423-44a3-4990-9e85-0239a1ee429a.png)

**Comment by FranckyC** [comment]
On the backlog but no ETA for now. Sorry for that.

**Reply by jameelrrs** [reply]
@FranckyC ,  This is one really useful feature to be added.  Thank you for the great work.

**Reply by aghyyy** [reply]
Hi @FranckyC , any update about this? Thank you!

**Reply by FranckyC** [reply]
Working on it. Can't tell when it will be ready however.

**Reply by wobba** [reply]
Also see https://github.com/microsoft-search/pnp-modern-search/pull/1342 for some discussion
